src/test/test-serialize.c

```diff
@@ -10,7 +10,7 @@
 #include "tests.h"
 #include "tmpfile-util.h"
 
-char long_string[LONG_LINE_MAX+1];
+static char long_string[LONG_LINE_MAX+1];
```

Comment:
This could also be done with `= { [0 ... LONG_LINE_MAX] = 'x', '\0'}` to statically initialize. Though, dunno how that affects binary size (if we care for that for test binaries).